addons/mail/static/src/models/call_settings_menu.js

```diff
@@ -116,6 +116,12 @@ registerModel({
             identifying: true,
             inverse: 'callSettingsMenu',
         }),
+        devices: many('CallSettingsMenuDevice', {
+            compute() {
+                return this.userDevices.map(webMediaDevice => ({ webMediaDevice }));
```

Comment:
strange to have 2 different names for the same concept `userDevice` here and `webMediaDevice` in child

Reply:
Current PR could rename field `userDevices` to something like `webMediaDevices`.
`webMediaDevice` is object from Web API for mode `CallSettingsMenuDevice`, which is a proper model.
`CallSettingsMenuDevice` wraps a `webMediaDevice` for effective working of features related to this model.